### PR TITLE
fix: add labels field in stream_route schema

### PR DESCRIPTION
**Pull request by Revolyssup**
### Description

Fixes multiple issues in ingress controller where the `labels` field is expected. This makes this resource consistent with other resource schemas.
Related issues:
 https://github.com/apache/apisix-ingress-controller/issues/2057
 https://github.com/apache/apisix-ingress-controller/issues/2023

### Checklist

- [ ] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: apisix/schema_def.lua [hunk 1]
```diff
@@ -892,6 +892,7 @@ _M.stream_route = {
         create_time = timestamp_def,
         update_time = timestamp_def,
         remote_addr = remote_addr_def,
+        labels = labels_def,
```
[the remark this answers is not in the document]
Comment (Revolyssup) [reply apisix/schema_def.lua:895]: This field is present in other resources as well. Like route and consumer.

[the remark this answers is not in the document]
Comment (monkeyDluffy6017) [reply apisix/schema_def.lua:895]: Is it necessary, can we fix this on the ingress side?

**Review by monkeyDluffy6017 — APPROVED** [review]

File: apisix/schema_def.lua (hunk 1 above)
Comment (monkeyDluffy6017) [inline apisix/schema_def.lua:895]: Please add comment here, like
```
The ingress project need this field
```

Comment (monkeyDluffy6017) [inline apisix/schema_def.lua:895]: Please add a comment here, like
```
The ingress project need this field
```

Comment (Revolyssup) [reply apisix/schema_def.lua]: done

**Review by monkeyDluffy6017 — APPROVED** [review]

**Review by AlinsRan — APPROVED** [review]

**Pull request merged**